src/main/java/core/basesyntax/Operations/SupplyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.Operations;
+
```

Comment:
There is a typo in the package name 'OpationStrategy'. It should likely be 'OperationStrategy'. Please correct the package name in the import statement to ensure proper functionality.